Atomic/syscontainers.py

```diff
@@ -1199,6 +1165,70 @@ def _do_checkout(self, repo, options):
 
         return options["values"]
 
+    def checkout_container_wrapup(self, rpm_install_content, tmpfiles_template, options, was_service_active):
+        """
+        Last stage of checking out a container. It includes outputting results,
+        setup for systemd part, and service starting
+        """
+        # Handle results based on rpm_install_content collected earlier
+        if rpm_install_content["rpm_file"]:
+            RPMHostInstall.install_rpm(rpm_install_content["rpm_file"])
+        else:
+            for installed_file in rpm_install_content["new_installed_files"]:
+                util.write_out("Created file {}".format(installed_file))
+
+        self._systemctl_command("daemon-reload")
+        if (tmpfiles_template):
+            self._systemd_tmpfiles("--create", options["tmpfilesout"])
+
+        # Note, when a rollback for container happens, the values from options will be reset to empty
+        self.checked_out_container_systemd_setup(options, was_service_active)
+
+    def checked_out_container_systemd_setup(self, options, was_service_active):
+        """
+        Handle container service accordingly based on was_service_active and cotnainer name.
```

Comment:
can we rename this function to something else?  What about `systemd_service_setup`?

Reply:
sure